src/main/java/com/mentalhealthapi/mentalhealthapi/dto/Blog.java

```diff
@@ -2,7 +2,8 @@
 
 import lombok.Data;
 
-public @Data class Blog {
+public @Data
+class Blog {
     private int id;
     private String disorderName;
     private String title;
```

Comment:
I think the @data declaration is better suited to be above the class. This is preference but also seems to be how the instances I've seen of this are laid out. 

https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3

Reply:
I will consider this!

Reply:
Leave this as it were.
- The IDE will automatically handle formatting for you, with Control-Alt-S.
- Line breaks do not decrease technical debt.